### PR TITLE
feat(angular): support Angular 15.2

### PR DESCRIPTION
**Pull request by Coly010**
Support Angular 15.2

### CODE REVIEW

File: packages/nx/src/utils/versions.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 export const nxVersion = require('../../package.json').version;
 
-export const angularCliVersion = '~15.1.0';
+export const angularCliVersion = '~15.2.0';
```
Comment (FrozenPandaz) [inline packages/nx/src/utils/versions.ts:3]: Sidenote: are we ready to remove this yet?

File: packages/workspace/src/utils/versions.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 export const nxVersion = require('../../package.json').version;
 
-export const angularCliVersion = '~15.1.0';
+export const angularCliVersion = '~15.2.0';
```
Comment (FrozenPandaz) [inline packages/workspace/src/utils/versions.ts:3]: Sidenote: are we ready to remove this yet?

**Review by FrozenPandaz — APPROVED** [review]

**Review by leosvelperez — APPROVED** [review]

File: packages/nx/src/utils/versions.ts (hunk 1 above)
Comment (Coly010) [reply packages/nx/src/utils/versions.ts:3]: I do believe that it is unused within Nx package. I'll double check

File: packages/workspace/src/utils/versions.ts (hunk 2 above)
Comment (Coly010) [reply packages/workspace/src/utils/versions.ts:3]: I'll check

**Pull request merged**